docs/README.md

```diff
@@ -0,0 +1,199 @@
+# @hoprnet/hopr-sdk
```

Comment:
Docs should be generated into the docs branch and do not make any commnits to main

Reply:
As commented in a meeting, we will create docs using gh-pages and in a different branch